### PR TITLE
Fix issues found from more complex usage

### PR DESCRIPTION
**Pull request by doanac**
* Improve README to match new fioctl
* Fix bad error handling logic to give better error response
* Fix bad logic for lodading client CAs

### CODE REVIEW

**Review by mwasilew — APPROVED** [review]
This works fine!

File: http_handlers.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func RegisterEchoHandlers(svcHandler ServiceHandler, e *echo.Echo) {
 			return c.String(http.StatusInternalServerError, err.Error())
 		}
 		bytes, err := validateRequest(svc, c)
-		if err != nil {
+		if bytes == nil { // validateRequest failed and sent the response
```
Comment (vkhoroz) [inline http_handlers.go:49]: Maybe `if bytes == nil || err != nil { ...`, just to cover all corner cases?

**Review by vkhoroz — APPROVED** [review]
LGTM

File: http_handlers.go (hunk 1 above)
Comment (doanac) [reply http_handlers.go:49]: fixed in force-push

**Pull request merged**